src/codegen/compvars.cpp

```diff
@@ -870,6 +921,13 @@ class IntType : public ConcreteCompilerType {
         return rtn;
     }
 
+    CompilerVariable* getiter(IREmitter& emitter, const OpInfo& info, VAR* var) override {
+        ConcreteCompilerVariable* converted = var->makeConverted(emitter, BOXED_INT);
+        CompilerVariable* rtn = converted->getiter(emitter, info);
+        converted->decvref(emitter);
+        return rtn;
```

Comment:
This kind of "defer to boxed/UNKNOWN" pattern is pretty common and is responsible for a lot of the boilerplate in this file :/  It's not important enough to be worth changing code once it's written, but I was happy with my experiment of having a default implementation that explicitly defers to UNKNOWN; see `compvars.h::_ValuedCompilerType<V>::unpack()`.